pkgs/development/libraries/libaudec/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub
+, libsndfile, libsamplerate
+, meson, ninja, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libaudec";
+  version = "0.2.4";
+
+  src = fetchFromGitHub {
+    owner = "zrythm";
+    repo = "libaudec";
+    rev = "v${version}";
+    sha256 = "1570m2dfia17dbkhd2qhx8jjihrpm7g8nnyg6n4wif4vv229s7dz";
+  };
+
+  buildInputs = [ libsndfile libsamplerate ];
+  nativeBuildInputs = [ meson ninja pkgconfig ];
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```
stdenv.lib got recently deprecated.